### ISSUE TITLE
ClimateType - smart14

### ISSUE THREAD

**Issue by sup-one** [Einstein42/udi-ecobee-poly#63]
Hi,

Just noticed that my Ecobee nodeserver logs are filling with the following error:

2020-12-03 03:19:46,857 Controller polyinterface      ERROR    node_types:getClimateIndex: Unknown climateType='smart14'
2020-12-03 03:22:46,806 Controller polyinterface      ERROR    node_types:getClimateIndex: Unknown climateType='smart14'
2020-12-03 03:25:46,884 Controller polyinterface      ERROR    node_types:getClimateIndex: Unknown climateType='smart14'
2020-12-03 03:28:46,792 Controller polyinterface      ERROR    node_types:getClimateIndex: Unknown climateType='smart14'

This entry is repeated exactly the same every 3 min

**Comment by jimboca** [comment]
Thanks, looks like something new added by Ecobee.  Will look into it.

**Comment by jimboca** [comment]
@sup-one is this still happening?  Looking at the code it seems that Ecobee is asking about that smart14 but it's not listed in the climate list they provide to me.

**Comment by sup-one** [comment]
@jimboca Yes, it is showing - I looked through the log when the nodeserver reboots and it corresponds to my custom comfort in Ecobee. 

Here is what I see:

2020-12-25 22:45:50,741 NodeServer polyinterface      DEBUG    ecobee-poly:start: customData=
{
  "_data_dtm": "2020-12-25T22:18:51.135691",
  "_data_lock": false,
  "climates": {
"XXXXXXXXXXX": [
      {
        "name": "Home",
        "ref": "home"
      },
      {
        "name": "Away",
        "ref": "away"
      },
      {
        "name": "Sleep",
        "ref": "sleep"
      },
      {
        "name": "Home Aft",
        "ref": "smart4"
      },
      {
        "name": "Home Winter",
        "ref": "smart11"
      },
      {
        "name": "Wknd",
        "ref": "smart13"
      },
      {
        "name": "Home Eve",
        "ref": "smart14"
      },
      {
        "name": "Wknd PM",
        "ref": "smart15"
      }
    ]
  },

**Comment by jimboca** [comment]
Ok I see, it should understand that. Can you download log package and email that to me?

**Comment by jimboca** [comment]
@sup-one Never mind, I found the issue.

**Comment by jimboca** [comment]
This will have to wait until next release after 2.2.0 is stable.  Looks like the number of Comfort settings is increased for some theremostat's so I'll need enhance to support this based on each thermostat.
